docs/proposals/20230612-node-os-image-caching.md

```diff
@@ -100,36 +99,38 @@ As an operator I would like to be able to have my node’s OS image cache for th
 
-Example of how the timestamp will be maintained in the AzureMachinePool:
+Example of how the timestamp will be maintained in the `AzureMachinePool`:
 
 ```yaml
 status:
-  lastPrototype: "2023-06-12T23:14:55Z"
+  lastImagePrototype: "2023-06-12T23:14:55Z"
 ```
 
-When the process is started it should go through the nodes of the cluster, choose a healthy node, shut it down, take a snapshot of it, restart it, create a compute image gallery image, delete the snapshot, and then configure the AzureMachinePool specs to use that compute image gallery image. After, it will store the current time as its timestamp. As a note, for the first implementation of this feature we will require the user to also use a compute image gallery image.
+When the process is started it should go through the nodes of the cluster, choose a healthy node, shut it down, take a snapshot of it, restart it, create a Azure Compute Gallery image, delete the snapshot, and then configure the `AzureMachinePool` spec to use that Azure Compute Gallery image. After, it will store the current time as its timestamp. As a note, for the first implementation of this feature we will require the user to also use a Azure Compute Gallery image.
 
 Diagram of the Node OS Caching Process:
 
 ![Figure 1](./images/node-os-image-cache.png)
 
-As for why the healthy node has to be shut down while creating a snapshot of it, if it isn’t shut down first then pods can be scheduled as the snapshot is taken which will cause some dangerous states in terms of how it exists after being utilized by the AzureMachinePools.
+As for why the healthy node has to be shut down while creating a snapshot of it, if it isn’t shut down first then pods can be scheduled as the snapshot is taken which will cause some dangerous states in terms of how it exists after being utilized by the `AzureMachinePools`.
 
-In terms of how a healthy node would be selected, there is already state data present on each AzureMachinePoolMachine which we can use and is listed in the examples below. An ideal node would be one which is running and healthy. Whichever node has been running and healthy for the longest amount of time should be chosen as it’s the most overall stable. This means that for AzureMachinePoolMachines we will take the node with the earliest creation time from metadata.creationTimestamp and has latestModelApplied : true present if it is an AzureMachinePoolMachine since this means that any user changes to the OS image have been rolled out already on this node. As the prototype is always from a successfully healthy and working node the image is always known to be working before being chosen for replication. In terms of knowing if the node os image has had updates that are not user made (so like automatic kernel patches), for the first development of this feature we won't know but all nodes in a cluster are configured to regularly automatically apply security patches to system packages.
+Healthy Node Selection:
```

Comment:
Formatting the headings like this might be a bit more readable.
```suggestion
#### Healthy Node Selection
```

Reply:
I agree/had been thinking about this, fixed!